src/components/search/autocomplete.component.tsx

```diff
@@ -42,7 +37,7 @@ const AutoComplete = ({ searchResults, style, setLocation }: IProps) => {
         renderItem={({ item }) => (
           <TouchableOpacity
             testID="touchableList"
-            onPressOut={() => setLocation(item)}
+            onPress={() => setLocation(item)}
```

Comment:
Tested on a Samsung S8, functionality appears unaffected with the touchableopacity.

Reply:
I said i was gonna throw in a comment about this on slack but forgot till now 😅

Reply:
I didn't, that issue occurred when using react-native's API, and it seems that gesture-handler's API is more reliable.

I'll put it back to onPressOut since it doesn't really affect me either way, but it doesn't work for me if we were to go back to react-native's TouchableOpacity.

Reply:
Thanks for testing @JimLau49 . @TheGreatMarkus Agreed, gesture-handler seems to be the recommended library to use for components with user interaction.